client/dashboard/sites/index.tsx

```diff
@@ -139,6 +140,13 @@ const DEFAULT_FIELDS: Field< Site >[] = [
 		},
 		enableSorting: false,
 	},
+	{
+		id: 'last_published',
+		label: __( 'Last published' ),
+		getValue: ( { item }: { item: Site } ) => item.options?.updated_at,
```

Comment:
My site-locked 100-year-plan site does not have this field set somehow, so we need to handle it, otherwise the column still throws the same 500 error when sorted 🥲
 
```suggestion
		getValue: ( { item }: { item: Site } ) => item.options?.updated_at ?? '',
```

<img width="1076" alt="Image" src="https://github.com/user-attachments/assets/ca6a122d-2a98-48dc-9d1c-9bc9aea4cabd" />

Reply:
Interesting, my 100-year-plan has one so I'll depend on you to see if the issue is fixed or not 🙂